packaging/nntrainer.spec

```diff
@@ -408,8 +413,8 @@ meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} \
       %{enable_profile} %{enable_nnstreamer_backbone} %{enable_tflite_backbone} \
       %{enable_tflite_interpreter} %{capi_ml_pkg_dep_resolution} \
       %{enable_reduce_tolerance} %{configure_subplugin_install_path} %{enable_debug} \
-      -Dml-api-support=enabled -Denable-nnstreamer-tensor-filter=enabled \
-      -Denable-nnstreamer-tensor-trainer=enabled -Denable-capi=enabled \
+      -Dml-api-support=enabled \
```

Comment:
I think nnstreamer related defines should be added. 
``` bash
%{enable_nnstreamer_tensor_trainer} %{enable_nnstreamer_tensor_filter}
```

Reply:
They were already in the line 412; that's why I removed these redundant lines.

Reply:
Thanks.